releases.yml

```diff
@@ -1,3 +1,9 @@
+- version: 3.12.2
```

Comment:
This version needs to be updated as well:
```suggestion
- version: 3.13.0
```

Reply:
Ow! Well spotted. Fixed that as well.